repository/kafka/docs/latest/upgrade.md

```diff
@@ -0,0 +1,77 @@
+# Upgrading the KUDO Kafka Operator
+
+
+
+KUDO upgrades work by linking the Kafka cluster Instance object to the correct operatorVersion object.
+
+We can have multiple operator versions in the same Kubernetes cluster. 
+One instance object represents a Kafka Cluster. And different instances can use same or different operator versions.
```

Comment:
Did you mean to add a line break or keep it as part of the line above? You'll have to add another line-break if you want it on new line.